src/index.js

```diff
@@ -48,6 +48,14 @@ class Masq {
       masqAppBaseUrl: options.masqAppBaseUrl ? options.masqAppBaseUrl : jsonConfig.masqAppBaseUrl,
       swarmConfig: options.swarmConfig ? options.swarmConfig : jsonConfig.swarmConfig
     }
+
+    this._reopenDbOnFocus()
```

Comment:
shouldn't we move `this._reopenDbOnFocus()` just after the call here https://github.com/QwantResearch/masq-lib/blob/0b6c1a885184e4e5d0dd3bbb178811dc27ba4867/src/index.js#L394 ?

Reply:
and what happens if we call `connectToMasq()` while the app is not authorized ?

Reply:
connectToMasq checks if a user is connected and if there is a connected user then it opens the db and starts the replication